BUILD

```diff
@@ -19,6 +19,7 @@ package_group(
         "//test/extensions/...",
         "//test/server",
         "//test/server/config_validation",
+        "//tools/extensions/...",
```

Comment:
maybe this should be more specific - not sure if the elipsis is necessary

Reply:
I'd make is as narrow as works.

Reply:
afaict this *is* as narrow as i can get it